ImP/preProcessing/testImageSimilarity.py

```diff
@@ -0,0 +1,27 @@
+import unittest 
+from preProcessor import PreProcessor 
+
+class ImageSimilarityTestCase(unittest.TestCase):
+	def setUp(self):
+		self.pathForTestImage = 'test_pictures/im1.JPG'
```

Comment:
Since this file could be run from elsewhere, you could use a pattern like the one I've implemented in other sub-modules where a class variable or settings file calculates and returns the directory of preProcessing. Then, you can do:
```
os.path.join(preProcessing_dir, 'im1.JPG')
```
The above would be safe no matter where the repo is located, or where this file is being called from.